featureflags/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 __author__ = """Enver Bisevac"""
 __email__ = "enver.bisevac@harness.io"
-__version__ = '1.3.0'
+__version__ = '1.3.1'
```

Comment:
1.3.0 has not been released, so you can just keep the number as is instead of bumping

Reply:
Cheers @andybharness